modules/nf-core/flye/tests/main.nf.test

```diff
@@ -0,0 +1,258 @@
+// According to the issue https://github.com/fenderglass/Flye/issues/164
+// Some fluctuations are expected because of the heuristics
+// Here we check the that test.assembly_info.txt contains at least one contig
```

Comment:
I love this

Reply:
To be fair, I only copied these lines from the original pytest 😅